README.md

```diff
@@ -85,6 +85,13 @@ building and testing a bit easier. The following actions are available through t
     Generate the project code documentation and make it viewable
     locally.
 
+To run locally:
+    1. In api.yml at x-runtime->stats->output (around line 9) change “DATADOG” to “NULL”
```

Comment:
Question: Any reason not to just change these to `NULL` by default, similar to what you've done in the docker-compose config?

Reply:
Honestly, I'm not sure. I decided not to push the api.yml changes because I didn't want to risk messing up anything relating to datadog in production.

Reply:
That makes sense. The api.yaml in this service is meant to be a functional example, not necessarily what we would run in production. It seems worth it, to me, to just drop DATADOG/metrics from api.yaml for the sake of making the compose environment buildable by default.

Reply:
Ah, okay. Didn't know that. I'll go ahead and remove those then. Thanks!

Reply:
Now that these items are already removed, this "run locally" sections can simply read: execute `make run`, or something to that effect.

Reply:
Oh yea... Silly mistake. Thanks for catching that.